nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -20,6 +20,12 @@
 
 - [SuiteNumérique Docs](https://github.com/suitenumerique/docs), a collaborative note taking, wiki and documentation web platform and alternative to Notion or Outline. Available as [services.lasuite-docs](#opt-services.lasuite-docs.enable).
 
+## Breaking Changes {#sec-releaase-25.11-breaking}
```

Comment:
Not my wheelhouse, but should we instead include the kOps notice under "Backward Incompatibilities"?